backend/frontend/templates/frontend/users_and_teams_nav/page.html

```diff
@@ -19,9 +19,8 @@ <h1 class="{{ styles.heading1 }} pb-2" @click="activeTab = 'users'">Teams</h1>
                     Show Users
                 </button>
             </div>
-            <form method="get" action="" class="mb-3 flex items-center gap-1">
-                {% csrf_token %}
-                <input type="text" name="search_term" placeholder="Search Teams..." id="search-term-box" hx-post="{% url 'view_partial_teams_list' %}" hx-trigger="keyup changed delay:500ms" hx-target="#teams-list" hx-swap="innerHTML" class="{{styles.input_small}} w-full lg:w-max">
+            <form method="get" action="" class="mb-3 flex items-center gap-1" hx-trigger="keydown" hx-on="keydown: event.key === 'Enter' ? event.preventDefault() : ''">
+                <input type="text" name="search_term" placeholder="Search Teams..." id="search-term-box" hx-get="{% url 'view_partial_teams_list' %}" hx-trigger="keyup changed delay:500ms" hx-target="#teams-list" hx-swap="innerHTML" class="{{styles.input_small}} w-full lg:w-max">
```

Comment:
We can use alter this behaviour with Alpinejs instead of HTMX like this: `x-on:keydown.prevent.enter=""`. I think this is more inline our current approach.